<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>843</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TotalComander</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>931</width>
      <height>791</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Directories</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="copy_dir">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Files</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="4">
      <widget class="QTreeView" name="Dirs">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="rename_dir">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="11">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="delete_dir">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QPushButton" name="copy_file">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="8">
      <widget class="QPushButton" name="rename_file">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QPushButton" name="create_file">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QPushButton" name="delete_file">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="4">
      <widget class="QListView" name="Files">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="create_dir">
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="11">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Book Antiqua</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>How to use:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDirectory">
    <property name="title">
     <string>Directory</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionCopy"/>
    <addaction name="actionRemove"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_2"/>
    <addaction name="actionCopy_2"/>
    <addaction name="actionRemove_2"/>
    <addaction name="actionDelete_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>Find</string>
    </property>
    <addaction name="actionFind_files"/>
   </widget>
   <addaction name="menuDirectory"/>
   <addaction name="menuFile"/>
   <addaction name="menuFind"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionCreate_2">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionCopy_2">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionRemove_2">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDelete_2">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionFind_files">
   <property name="text">
    <string>Find files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
